braindecode/datasets/base.py

```diff
@@ -56,6 +49,16 @@ def __getitem__(self, index):
     def __len__(self):
         return len(self.raw)
 
+    def _create_description(self, description):
+        if description is not None:
+            if (not isinstance(description, pd.Series)
+                    and not isinstance(description, dict)):
+                raise ValueError(f"'{description}' has to be either a "
+                                 f"pandas.Series or a dict")
+            if isinstance(description, dict):
+                description = pd.Series(description)
+        return description
```

Comment:
this function does not do anything with self so it should not be a class method
but rather a private function in the file